code/src/OSFramework/DrawingTools/ITool.ts

```diff
@@ -8,15 +8,19 @@ namespace OSFramework.DrawingTools {
         drawingTools: IDrawingTools;
         isReady: boolean;
         map: OSMap.IMap; //IMap
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        options: any;
```

Comment:
Please add a TODO here as well, change this any to the structure that you will create in the future (global d.ts)